src/librustc_trans/cabi_powerpc.rs

```diff
@@ -28,11 +28,7 @@ fn align(off: usize, ty: Type) -> usize {
 
 fn ty_align(ty: Type) -> usize {
     match ty.kind() {
-        Integer => {
-            unsafe {
-                ((llvm::LLVMGetIntTypeWidth(ty.to_ref()) as usize) + 7) / 8
-            }
-        }
+        Integer => ((ty.int_width() as usize) + 7) / 8,
```

Comment:
I forgot to mention but these two are drive by cleanups. The int_width method is equivalent to the llvm call that was deleted.